### ISSUE TITLE
lint: warn on unverified error

### ISSUE THREAD

**Issue by cristaloleg** [go-critic/go-critic#432]
Name: `???`
Before:
```go
a, err := foo()
if err != nil { ... }
b, err := bar()    // err wasn't checked
baz()
```
After:
```go
a, err := foo()
if err != nil { ... }
b, err := bar()
if err != nil { ... }
baz()
```

Suggested by @DmitriyMV

**Comment by Quasilyte** [comment]
Sounds good to me.

**Comment by DmitriyMV** [comment]
Unverified -> Unchecked

**Comment by DmitriyMV** [comment]
Was this fixed?

**Comment by cristaloleg** [comment]
@DmitriyMV nope. sorry.

**Comment by quasilyte** [comment]
Doesn't errcheck catch this?

**Comment by Antonboom** [comment]
https://golangci-lint.run/usage/linters/#errcheck
